test/unit/models/index.test.ts

```diff
@@ -133,4 +133,133 @@ describe("test installation model", () => {
 			expect(subscription.jiraHost).toBe(renamedInstallPayload.baseUrl);
 		}
 	});
+
+	describe("duplicate entries in Installations table", () => {
+		it("should return no installations if jiraHost is not found", async () => {
+			const jiraHost = "https://myfakejirasite.net";
+			const installations = await Installation.getAllForHost(jiraHost);
+			const installation = await Installation.getForHost(jiraHost);
+
+			expect(installations.length).toEqual(0);
+			expect(installation).toEqual(null);
+		});
+
+		it("should return a single installation if 1 host is installed", async () => {
+			await Installation.install({
```

Comment:
Since we're testing the 2 functions you changed and not the `install` function.  I suggest you change all the `install` functions to `create` instead.  That way it's "closer to the metal" and should be better to avoid side effects.